web/frontend/src/components/arm/index.svelte

```diff
@@ -249,172 +249,159 @@ const armCopyJoints = () => {
   }, {})));
 };
 
+onStop?.(stop)
```

Comment:
Why isn't this call to `onStop` declared reactively? Should it be reactive, since `onStop` is a prop? Or is this something that would be better served by a context?

Reply:
Ah yes, it probably should be reactive at the least. Now that I think about it, making it a context would probably alleviate some of the awkwardness concerns @ethanlook has. I'll change this to a context now!